<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>FormCreateMatch</class>
 <widget class="QDialog" name="FormCreateMatch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create match</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupMap" >
     <property name="title" >
      <string>Map</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboMap" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>This map will be played.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonMapSelect" >
        <property name="whatsThis" >
         <string>Click here to get more details of the maps.</string>
        </property>
        <property name="text" >
         <string>Details</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Players</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QListView" name="listPlayers" >
        <property name="whatsThis" >
         <string>Select the players who want to play.&lt;br>Doubleclick on a player to change his/her name.</string>
        </property>
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonAdvanced" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Advanced</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>231</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FormCreateMatch</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>FormCreateMatch</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>418</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel" >
     <x>282</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
